src/Console/Command/WorkerCommand.php

```diff
@@ -139,6 +139,7 @@ private function runWorker(
             $processResult = $this->applicationFileProcessor->processFiles(
                 $filePaths,
                 $configuration,
+                null,
```

Comment:
When processing files from the worker, we set the allFiles parameter to null so that this list of files is loaded from the cache